src/bokeh/models/formatters.py

```diff
@@ -654,6 +654,13 @@ def __init__(self, *args, **kwargs) -> None:
     as they are on boundary of the days scale: ["06/08", "06h", "12h", 18h", "06/09"]
     """)
 
+    hide_repeats = Bool(default=False, help="""
+    Whether repeated formatted tick values will be suppressed. For example, an initial set of ticks ["06/07", "06/07",
+    "06/07", "06/08", "06/08"] will become ["06/07", "", "", "06/08", ""]. Only the base label, without any additional
+    context, is considered when determining repeats. (If the context itself is a DateTimeTickformatter, then this
+    property may also be set for the context separately, if desired).
+    """)
```

Comment:
I would reformat this to make this block of text less overwhelming and better match other docstrings:
```suggestion
    hide_repeats = Bool(default=False, help="""
    Whether repeated formatted tick values will be suppressed.

    For example, an initial set of ticks ``["06/07", "06/07", "06/07", "06/08",
    "06/08"]`` will become ``["06/07", "", "", "06/08", ""]``. Only the base
    label, without any additional context, is considered when determining
    repeats. If the context itself is a ``DateTimeTickformatter``, then this
    property may also be set for the context separately, if desired.
    """)
```